proj.sh

```diff
@@ -36,12 +37,13 @@ function clean-stack {
 }
 
 function logs {
-  docker compose -f docker-compose.yml -f docker-compose-local.yml logs -f $@
+  docker compose --env-file $CHOSEN_ENV -f docker-compose.yml -f docker-compose-local.yml logs -f $@
 }
 
 function deploy-stack {
+  CHOSEN_ENV="env-${1:=local}.env"
```

Comment:
The `$CHOSEN_ENV` var should be reset to "local" after deploying (regardless if the deploy is successful or not). Or else the other commands such as `clean-stack` or `import-db` will run on the run context. Not sure what's the best way to go about this.

Reply:
Also we should really update the `clean-stack` command so it doesn't remove all docker containers. @mihai-sysbio do you know if we have an existing issue for this? I can create one otherwise.

Reply:
Fixed in https://github.com/MetabolicAtlas/MetabolicAtlas/pull/794/commits/ca9ac041c9351f46f1825e3676243dfbc26cf1a7

Reply:
Thanks a lot for the fix. I see that we are referring to `local` in a few places in that file, which isn't so nice. I'm okay with leaving it be though.

Reply:
Definitely room for improvement. I'm fine with leaving it like this now as well.

@mihai-sysbio just wanted to ping you about [this comment](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/794#discussion_r768790064) in case you missed it. Would be nice to get your input.

Reply:
Ah thanks for the reminder, I missed that.
If I understand correctly, you're proposing to keep the `clean-stack` command, but to improve its behavior so that it removes only project-related containers, instead of all containers?

Reply:
> If I understand correctly, you're proposing to keep the `clean-stack` command, but to improve its behavior so that it removes only project-related containers, instead of all containers?

Yes, that's right.

Reply:
@mihai-sysbio To follow up on this. Did we agree that there is no current issue for this and a new one should be created? Also as mentioned yesterday, please feel free to merge this PR even before the new issue is created (if necessary).

Reply:
#816 was create to address this, as the behavior of this command was neither introduced nor altered with this PR.